crates/eth-rpc/src/lib.rs

```diff
@@ -24,7 +24,7 @@ pub enum RpcError {
 ///
 /// Will return `Err` if an error occurs when running the `ServerBuilder` start fails.
 pub async fn run_server(
-    starknet_client: Box<dyn KakarotProvider>,
```

Comment:
oops, typo here from former pr, it should be `starknet_client`  -> `kakarot_client`, right?

Reply:
i can do it